test/helper/fake_projects.go

```diff
@@ -0,0 +1,103 @@
+package helper
```

Comment:
How about to create a directory with mocks? E.g. `test/mock/projects.go`

Reply:
Moved to `pks/utils/atlastest` as it will need to be accessed by unit tests under ´pkg/´ as well.